webpack.config.js

```diff
@@ -104,12 +104,6 @@ var build = {
   resolve: {extensions: ['', '.js']},
   stats: {colors: true},
   externals: {
-    flux: {
```

Comment:
If redux has bower package, than we should also make it external. If not, then we need to publish it to bower =)